rpms/quattor-development.pan

```diff
@@ -22,6 +22,8 @@ unique template rpms/quattor-development;
   pkg_repl('perl-Taint-Runtime');
   pkg_repl('perl-enum');
   pkg_repl('perl-XML-Simple');
+  pkg_repl('perl-Module-Load');
+  pkg_repl('perl-Perl-Critic');
```

Comment:
it needs a version that is more recent than epel has. i'm going to add the correct version to the externals repo asap.